api/routes.js

```diff
@@ -20,6 +21,8 @@ Router.post('/sign/out', wrap(SignUp.signout, SignUp));
 
 // Plan
 Router.get('/plan/:id', wrap(Plan.read, Plan));
+Router.get('/plan/:id/subscribe', wrap(Subscription.subscribe, Subscription));
```

Comment:
Great, butter to use PUT or POST for security reasons

Reply:
DONE